src/io/io_routines.f90

```diff
@@ -689,7 +695,7 @@ subroutine io_write4d(filename,varname,data_out, dimnames)
 
         ! Open the file. NF90_NOWRITE tells netCDF we want read-only access to
         ! the file.
-        call check( nf90_create(filename, NF90_CLOBBER, ncid), filename)
+        call check( nf90_create(filename, NF90_HDF5, ncid), filename)
```

Comment:
and again, so maybe make opening files into a subroutine so you don't repeat yourself